index.html

```diff
@@ -90,7 +90,234 @@ <h1 class="primary-text-header">I’m Steven Levoyer to see you!</h1>
+                            project you need coded, don’t hestiate to contact me.</p>
+                        <div class="social-media-about-me">
+                            <div class="suppoting-text-about-me">
+                                <a href="#" class="lets-connect">LET’S CONNECT</a>
+                            </div>
+                            <div class="frame-1">
+                                <ul class="header-icons">
+                                    <li><a href="#"><img src="./img/twitter.svg"></a></li>
+                                    <li><a href="#"><img src="./img/Linkedin.svg"></a></li>
+                                    <li><a href="#"><img src="./img/m.svg"></a></li>
+                                    <li><a href="#"><img src="./img/github.svg"></a></li>
+                                    <li><a href="#"><img src="./img/5.svg"></a></li>
+                                </ul>
+                            </div>
+                        </div>
+                        <button type="button" class="action-button">
+                            <p class="text-a">Get my resume</p>
+                        </button>
+                    </div>
+                    <ul class="skill-list">
+                        <li>
+                            <div class="frame-block">
+                                <div id="frame-skill">
+                                    <span class="text-skill">Languages</span>
+                                    <button type="button" class="button-skill"><img src="./img/V.svg" class="V"></button>
+                                    <div class="separator" id="s-1"></div>
+                                </div>
+                                <div class="languages-list">
+                                    <div class="frame-list">
+                                        <img class="frame-img" src="./img/Ellipse 1.svg">
+                                        <span class="frame-text">javaScript</span>
+                                    </div>
+                                    <div class="frame-list">
+                                        <img class="frame-img" src="./img/Ellipse 1-1.svg">
+                                        <span class="frame-text">HTML</span>
+                                    </div>
+                                    <div class="frame-list">
+                                        <img class="frame-img" src="./img/Ellipse 1-2.svg">
+                                        <span class="frame-text">CSS</span>
+                                    </div>
+                                </div>
+                            </div>
+                        </li>
+                        <li>
+                            <div class="frame-block">
+                                <div id="frame-skill">
+                                    <span class="text-skill">Frameworks</span>
```

Comment:
- [ ] Please change these icons from down arrow to an arrow facing the right. THis is to match the figma template.
### Your design
![2](https://user-images.githubusercontent.com/94551484/213791797-4171461b-db01-4685-8a84-020d291c3c80.png)
### Figma
![f2](https://user-images.githubusercontent.com/94551484/213791802-8fb47516-7441-4db6-911b-6e416467ced8.png)